app.rb

```diff
@@ -0,0 +1,126 @@
+require_relative('./person')
+require_relative('./book')
+require_relative('./rental')
+require_relative('./student')
+require_relative('./teacher')
+require 'date'
+
+class Methods
+  def initialize
+    @rentals = []
+    @people = []
+    @books = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'No book added yet'
+      return
+    end
+    @books.map do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'No person record created yet'
+      return
+    end
+    @people.map do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'That is not a valid input'
+      return
+    end
+
+    puts 'Person created successfully'
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    student = Student.new(age, name)
+    @people.push(student)
+    puts 'Student Created Successfully'
+  end
```

Comment:
- [ ] While creating students ask 'has parent permission' like in the video.

![image](https://user-images.githubusercontent.com/38635158/163622352-c88b4943-0524-400e-82b2-c95e707456c3.png)